src/main/java/com/airdnb/clone/domain/reservation/ReservationRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.airdnb.clone.domain.reservation;
+
+import com.airdnb.clone.domain.reservation.entity.Reservation;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ReservationRepository extends JpaRepository<Reservation, Long> {
+}
```

Comment:
```suggestion
@Repository
public interface ReservationRepository extends JpaRepository<Reservation, Long> {
}
```

명확성을 위해 `@Repository` 애너테이션을 붙여볼까요